accelerator/core/apis.c

```diff
@@ -78,6 +78,11 @@ status_t api_find_transaction_objects(const iota_client_service_t* const service
     goto done;
   }
 
+  hash243_queue_entry_t* tmp_queue_entry = NULL;
+  while (hash243_queue_count(req->hashes) > FIND_TXN_LIMITATION) {
+    tmp_queue_entry = hash243_queue_pop(&req->hashes);
```

Comment:
What is the `tmp_queue_entry` for?
It seems that it stores the pop out value but would not be used?
If so, just remove it.